ansible-tests/validations/library/pacemaker.py

```diff
@@ -0,0 +1,42 @@
+import itertools
+from subprocess import Popen, PIPE
+from xml.etree import ElementTree
+
+from ansible.module_utils.basic import *
+
+
+def parse_pcs_status(pcs_status_xml):
+    root = ElementTree.fromstring(pcs_status_xml)
+    result = {
+        'failures': [dict(failure_element.items()) for failure_element
+                     in root.findall('failures/failure')],
+    }
+    return result
+
+
+def format_failure(failure):
```

Comment:
Thanks! I'm not quite sure what you mean here, though.

Do you mean keeping `task`, `op_key`, `node` etc. and building the `failures` in `parse_pcs_status` using those keys explicitly instead of doing `dict(failure_element.items())`?